package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "atlas-ui-framework",
-    "version": "2.6.3",
+    "version": "2.6.4",
```

Comment:
This should not be changed yet. We only change it just before we release

Reply:
✅